rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline_segmentation.py

```diff
@@ -0,0 +1,531 @@
+# 1D labels array in a batch
+        self.labels = np.zeros(self.count_batch, dtype="int32")
+        self.loader.GetBBLabels(self.labels)
+        # print(self.labels)
+# 1D bboxes array in a batch
+        self.bboxes = np.zeros((self.count_batch*4), dtype="float32")
+        self.loader.GetBBCords(self.bboxes)
+        # print("BBoxes (rocAL): ",self.bboxes)
+#Image sizes of a batch
+        self.img_size = np.zeros((self.bs * 2),dtype = "int32")
+        self.loader.GetImgSizes(self.img_size)
+        # print("Image sizes:", self.img_size)
+#Image ROI width and height
+        self.roi_width = np.zeros((self.bs),dtype = "uint32")
+        self.roi_height = np.zeros((self.bs),dtype = "uint32")
+        self.loader.getOutputROIWidth(self.roi_width)
+        self.loader.getOutputROIHeight(self.roi_height)
+        self.roi_sizes = np.vstack((self.roi_height,self.roi_width)).T # Old
+        self.roi_sizes_wh = np.vstack((self.roi_width,self.roi_height)).T # New
+#Mask info of a batch
+        self.mask_count = np.zeros(self.count_batch, dtype="int32")
+        self.mask_size = self.loader.GetMaskCount(self.mask_count)
+        self.polygon_size = np.zeros(self.mask_size, dtype= "int32")
+        self.mask_data = np.zeros(100000, dtype = "float32")
+        self.loader.GetMaskCoordinates(self.polygon_size, self.mask_data)
+
+        count =0
+        sum_count=0
+        j = 0
+        list_poly = []
+        iteration1 = 0
+        iteration = 0
+        self.target_batch = []
+        self.roi_image_size = []
+        self.roi_image_size_wh = []
+        for i in range(self.bs):
+            count = self.bboxes_label_count[i]
+            self.img_name = self.Img_name[i*16:(i*16)+12]
+            self.img_name=self.img_name.decode('utf_8')
+            self.img_name = np.char.lstrip(self.img_name, chars ='0')
+            self.img_size_2d_numpy = (self.img_size[i*2:(i*2)+2])
+            self.img_roi_size2d_numpy = (self.roi_sizes[i])
+            self.img_roi_size2d_numpy_wh = (self.roi_sizes_wh[i])
+            # Image Size ROI
+            roi_tmp_list = self.img_roi_size2d_numpy.tolist()
+            self.roi_image_size.append(torch.Size(roi_tmp_list))
+            roi_tmp_list = self.img_roi_size2d_numpy_wh.tolist()
+            self.roi_image_size_wh.append(torch.Size(roi_tmp_list))
+            self.label_2d_numpy = (self.labels[sum_count : sum_count+count])
+            self.bb_2d_numpy = (self.bboxes[sum_count*4 : (sum_count+count)*4])
+
+            # print("\nBefore : self.bb_2d_numpy\n", self.bb_2d_numpy)
+            for index, element in enumerate(self.bb_2d_numpy):
+                if index % 2 == 0:
+                    self.bb_2d_numpy[index] = self.bb_2d_numpy[index] * self.img_roi_size2d_numpy_wh[0]
+                elif index % 2 != 0:
+                    self.bb_2d_numpy[index] = self.bb_2d_numpy[index] * self.img_roi_size2d_numpy_wh[1]
+
+            # print("\nAfter : self.bb_2d_numpy\n", self.bb_2d_numpy)
```

Comment:
Remove the print statemnts that are commented out

Reply:
Resolved in the latest commit